### DISCUSSION TITLE
netflix conductor server as spring boot app

### DISCUSSION THREAD

**Discussion by laxmikantb** [Netflix/conductor#2623] [Q&A] [answered]
Hi, Does anybody tried to run server as spring boot app.

Would like instantiate conductor , i am trying use latest 3.3.6 version. also where i can find server-all.har for this 3.3.6 version?

your help will be appreciated

**Comment by apanicker-nflx** [comment, marked as answer]
The conductor server is a spring boot application and can be run from the command line using `./gradlew bootRun` [Documentation](https://netflix.github.io/conductor/architecture/#start-local-server)

**Comment by autodidactic** [comment]
hi 
i am having issues running this with java 16 and gradle 6.8.3 version . any fix for making it working with higher java version?

**Reply by apanicker-nflx** [reply]
@autodidactic We have not tested running Conductor with java 16, so cannot comment on what changes might be needed, if any, at this time. 

Additionally, as per the [Gradle Java Compatibility Matrix](https://docs.gradle.org/current/userguide/compatibility.html) you would need atleast gradle 7.0 to build a java 16 project. If you wish to use java 16, we would recommend you to please fork the project and make changes as necessary.

**Comment by autodidactic** [comment]
Ok thanks
I will try to rebuild with lower Java version since it’s easier than upgrading grade build
